benchmarks/cache_benchmarks/database_cache/benchmark.py

```diff
@@ -0,0 +1,73 @@
+import random
+
+from django.core.cache import caches
+from django.core.management import call_command
+
+from ...utils import bench_setup
+
+
+class DatabaseCacheBackend:
```

Comment:
Do you think we could add coverage for the some of the other backends too? The benchmarks would be the same, it's just a different setting? 🤔

Reply:
Can we do this in separate pull request?